helpers/combine_docfiles.py

```diff
@@ -54,6 +54,10 @@
 replace_content = groups[0] + groups[1]
 
 # Find where to put the replacement content, overwrite the input file
-groups = re.match(insert_separator_regex, input, re.DOTALL).groups(0)
+m = re.match(insert_separator_regex, input, re.DOTALL)
```

Comment:
The bug related to this change has been recorded here https://github.com/terraform-google-modules/terraform-google-module-template/issues/16. The workaround is to ensure that the license comment is present in all Terraform configuration files.

Reply:
Heh, the missing file is one unrelated to this CL, that's why I changed the script instead. I'll change the unrelated file and revert the script, thanks.

Reply:
I fail to see how this is relevant, as the exception is raised in `./codelabs/simple/README.md`.

Reply:
@ludoo It's fine if you leave as-is.

Reply:
Yup, I reverted the change, checks are fine. Feel free to approve. ;)